app/presenters/saml_request_presenter.rb

```diff
@@ -0,0 +1,64 @@
+class SamlRequestPresenter
```

Comment:
That class will be going away entirely since it all it does is call the scoper class. That will be either in a separate PR, or later on in this PR once I get all tests to pass, and once people have had a chance to review it.